sdk/log/simple.go

```diff
@@ -18,11 +18,12 @@ type SimpleProcessor struct {
 
 // NewSimpleProcessor is a simple Processor adapter.
 //
-// This Processor is not recommended for production use. The synchronous
-// nature of this Processor make it good for testing, debugging, or
-// showing examples of other features, but it can be slow and have a high
-// computation resource usage overhead. [NewBatchProcessor] is recommended
-// for production use instead.
+// This Processor is not recommended for production use due to its synchronous
```

Comment:
Just sharing some wording suggestion: 
This is not fit for production use, **due to lack of batching. i.e it sends batch of size1**, so the transport (http/grpc/tcp) overhead is too much. And this is only valid if the transport has "setup" overhead, so it makes sense to send a bunch of items in one go. This is the case for OTLP over gRPC/HTTP.

For transports like etw etc. there is negligible overhead, so batch of size 1 is perfectly fine.